opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -340,19 +340,24 @@ message Int64DataPoint {
   // The set of labels that uniquely identify this timeseries.
   repeated opentelemetry.proto.common.v1.StringKeyValue labels = 1;
 
-  // start_time_unix_nano is the time when the cumulative value was reset to zero.
-  // This is used for Counter type only. For Gauge the value is not specified and
-  // defaults to 0.
+  // start_time_unix_nano is the last time when the value was reset to "zero".
+  // For some metric types this is ignored, see MetricsDescriptor types for more
+  // details. 
   //
-  // The cumulative value is over the time interval (start_time_unix_nano, time_unix_nano].
-  // Value is UNIX Epoch time in nanoseconds since 00:00:00 UTC on 1 January 1970.
+  // The aggregation value is over the time interval (start_time_unix_nano,
+  // time_unix_nano].
+  // 
+  // Value is UNIX Epoch time in nanoseconds since 00:00:00 UTC on 1 January
+  // 1970.
   //
-  // Value of 0 indicates that the timestamp is unspecified. In that case the timestamp
-  // may be decided by the backend.
+  // Value of 0 indicates that the timestamp is unspecified. In that case the
+  // timestamp may be decided by the backend.
   fixed64 start_time_unix_nano = 2;
 
-  // time_unix_nano is the moment when this value was recorded.
-  // Value is UNIX Epoch time in nanoseconds since 00:00:00 UTC on 1 January 1970.
+  // time_unix_nano is the moment when this aggregation value was reported.
```

Comment:
Why change this to "reported"? To me that sounds like it's the time you send the request. Isn't "recorded" more correct?

Reply:
I think it's to be consistent with `ReportStrategy` in #192. I agree though, I thought the same thing

Reply:
My problem with "recorded" was that it hints (maybe hidden hint) that this is the time when the measurement was recorded using the SDK instruments, which is not the case